logging/src/main/java/org/jboss/as/logging/filters/FilterResourceDefinition.java

```diff
@@ -0,0 +1,250 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ *
+ * Copyright 2019 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.as.logging.filters;
+
+import static org.jboss.as.logging.CommonAttributes.CLASS;
+import static org.jboss.as.logging.CommonAttributes.MODULE;
```

Comment:
@jamezp I reviewing this I noticed the definitions of these is incorrect, at least for this use case and the other ones I could see. So I filed https://issues.jboss.org/projects/WFCORE/issues/WFCORE-4631. Not necessary to correct in this PR as it is a pre-existing issue.

Reply:
That seems correct. I'll get a fix for that probably today.